### ISSUE TITLE
TeX drawing inside equations

### ISSUE THREAD

**Issue by dginev** [brucemiller/LaTeXML#1244]
We should ideally have a decent support for visual snippets inside equations (see StackOverflow link below for a mathematically reasonable example). This requires non-obvious interplay between MathML and SVG, which may in turn require the specifications to provide additional guidance.

Thanks to Peter K Sweby for requesting the feature on the mailing list.

Example 1:
```tex
\[
w(x,y)=\mbox{\tikz{\draw[line width=1pt] (0,0) rectangle(30pt,40pt);}}.
\]
```
Example 2:
![image](https://user-images.githubusercontent.com/348975/76262542-0251f080-6233-11ea-92d4-af3330327aae.png)

https://tex.stackexchange.com/questions/179035/tikz-figure-inside-equation
```tex
\documentclass{article}
\usepackage{tikz}

\def\straight{\tikz[baseline=.1ex]{
\fill (0,0) circle (1pt) coordinate (A);
\fill (0,1.5ex) circle (1pt) coordinate (B);
\draw (A)--(B);}
}

\def\curved{\tikz[baseline=.1ex]{
\fill (0,0) circle (1pt) coordinate (A);
\fill (0,1.5ex) circle (1pt) coordinate (B);
\useasboundingbox (0,0) rectangle (.7ex, 1.5ex);
\draw (A) .. controls +(1ex,-1ex) and +(1ex, 1ex) .. (B);}%
}

\begin{document}

\[ P(\straight) + P(\curved) = p \]

\end{document}
```

**Comment by brucemiller** [comment]
Not sure when this was last tried, but these 2 examples work for me, at least to the extent of embedding tikz svg into math (the 2nd examples are missing the connecting lines, but they do that whether embedded in math or not --- ie. a separate tikz bug).

**Comment by dginev** [comment]
Interesting, the examples are indeed "modestly successful" with the latest master. The first question was to even ask what is the best way to stack MathML and SVG together, but it sounds like you've thought about this before? The markup generated is an `<mtext>` element that hosts the `<svg>` drawing, which is probably the best MathML 3 has to offer.

If that is the case, this may be closeable already - I had forwarded the issues as a question from the mailing list, to which I didn't personally have an answer to at the time.

**Comment by brucemiller** [comment]
A quick experiment in the inspector, it looks like the connecting paths only missing a `stroke` attribute (or got nested wrong in the `svg:g`).  Possibly an easy fix?

**Comment by dginev** [comment]
Resolved in its basic concern via #1413 , thanks Bruce!
